pkg/geodata/geodata.go

```diff
@@ -227,6 +230,77 @@ AND nomis_category.year = %d
 	return app.collectCells(ctx, sql)
 }
 
+// polygonQuery returns the csv table for areas within radius meters from location
+//
+func (app *Geodata) polygonQuery(ctx context.Context, polygon string, geotypes, cats []string) (string, error) {
+	points, err := ParsePolygon(polygon)
+	if err != nil {
+		return "", fmt.Errorf("parsing polygon: %q: %w", polygon, err)
+	}
+
+	template := `
+SELECT
+    geo.code AS geography_code,
+    nomis_category.long_nomis_code AS category_code,
+    geo_metric.metric AS value
+FROM
+    geo,
+    geo_type,
+    geo_metric,
+    data_ver,
+    nomis_category
+WHERE ST_Covers(
+    ST_Polygon(
+        'LINESTRING (%s)'::geometry,
+        4326
+    ),
+    geo.wkb_geometry
+)
+AND geo.valid
+AND geo_type.id = geo.type_id
+%s
+AND geo_metric.geo_id = geo.id
+AND data_ver.id = geo_metric.data_ver_id
+AND data_ver.census_year = %d
+ANd data_ver.ver_string = '2.2'
```

Comment:
OOS for this, but should this data_ver value be pulled from config / the query?

Reply:
Yes. I guess we can add options when we eventually add more versions.